Utils/request_contributor_review.py

```diff
@@ -187,20 +221,138 @@ def check_reviewers(reviewers: set, pr_author: str, version: str, modified_files
+        )
+
+
+def send_email_to_reviewers(reviewers_emails: str, refresh_token: str, pack_name: str,
+                            pr_number: str):
+    """ Compose mail and send it to the reviewers_emails, to review the changes in their pack
+
+    Args:
+        reviewers_emails(str): reviewers of the pack to send mail to them
+        refresh_token(str): refresh token to send mails using gmail API
+        pack_name(str): pack that was modified
+        pr_number(str): github pr number
+
+    """
+    access_token = get_access_token(refresh_token)
+    credentials = AccessTokenCredentials(access_token, 'Demisto Github send mails to contributors')
+    service = build('gmail', 'v1', credentials=credentials)
+
+    email_content = f"Changes were made to the {pack_name} content pack that you contributed. \n"
+    email_content += f"Please review the changes in the following files in the contribution "
+    email_content += f"[Pull Request](https://github.com/demisto/content/pull/{pr_number}/files)."
+
+    email_subject = f'Changes made to {pack_name} content pack'
+
+    message = MIMEText(email_content, 'plain', 'utf-8')
+    message['bcc'] = reviewers_emails  # send mails to all contributors in bcc
+    message['from'] = EMAIL_FROM
+    message['subject'] = email_subject
+    message_to_send = {'raw': base64.urlsafe_b64encode(message.as_bytes()).decode()}
+
+    try:
+        service.users().messages().send(userId=EMAIL_FROM, body=message_to_send).execute()
+        print(f'Email sent to {reviewers_emails} reviewers of pack {pack_name}')
+    except errors.HttpError as e:
+        print(f'An error occurred during sending emails to contributors: {str(e)}')
+        sys.exit(1)
+
+
+def get_access_token(refresh_token: str):
+    """ Gets access token from os environment if it was saved there. Else, generates access token from refresh token.
+
+    Args:
+        refresh_token(str): refresh token to obtain access token, to send mails using gmail API
+
+    Returns:
+        access_token(str): access token is used to send mails using gmail API
+
+    """
+    access_token = os.getenv('ACCESS_TOKEN')
+    valid_until = int(os.getenv('VALID_UNTIL')) if os.getenv('VALID_UNTIL') else None
+
+    # check if access token is valid
+    if access_token and valid_until:
+        if int(datetime.now().timestamp()) < valid_until:
+            return access_token
+
+    if not refresh_token:
+        print(f"Error obtaining access token. Failed sending mails.")
+        sys.exit(1)
+
+    # else access token should be obtained from refresh token
+    http_client = httplib2.Http()
+    body = {
+        'refresh_token': refresh_token,
+        'client_id': GMAIL_CLIENT_ID,
+        'grant_type': 'refresh_token',
+    }
+    resp, content = http_client.request(TOKEN_URL, "POST", urllib.parse.urlencode(body), TOKEN_FORM_HEADERS)
+
+    if resp.status not in [200, 201]:
+        print(f"Error obtaining access token. Failed sending mails.")
```

Comment:
let's adda the error
```suggestion
        print(f"Failed to send emails: error obtaining access token\n{content}.")
```